draft-thomson-webpush-protocol.xml

```diff
@@ -364,7 +407,71 @@ Location: https://push.example.net/d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
         Push services MUST NOT generate a 413 status code in responses to an
         entity body that is 4k (4096 bytes) or less in size.
       </t>
+      <section anchor="request_receipt" title="Receiving Push Message Receipts">
+        <t>
+          An application server can use the Receipt header field to request a
+          confirmation from the push server when a push message is delivered and
+          acknowledged by the user agent. The Receipt header field value is the
+          "receipt" URI created when the application server subscribed to
+          receipts.
+        </t>
+        <figure>
+          <artwork type="abnf">
+            <![CDATA[
+Receipt = URI-reference
+          ]]>
+          </artwork>
+        </figure>
+        <t>
+          The application server requests the delivery of receipts from
+          the push server by making a HTTP GET request to the "receipt"
+          resource. The push service does not respond to this request, it instead
+          uses <xref target="I-D.ietf-httpbis-http2">HTTP/2 server push</xref>
+          to send push receipts when messages are acknowledged (<xref
+          target="acknowledge_message"></xref>) by the user agent.
+        </t>
+        <t>
+          Each receipt is pushed in response to a synthesized GET request.
+          The GET request is made to the same message resource that was created
+          by the push server when the application server requested message
+          delivery. The message resource is set in the :path pseudo header.
+          The response body is empty.
```

Comment:
We need to define what the response looks like.  I'm going to suggest that this is simply 410 (Gone).